### PR TITLE
Add a section to the FAQ about getting package release notifications

### LINKED ISSUE #98: Link to release notifications help on project detail page
This would include the ability to upload a requirements.txt adn get notified for everything in it.

-------
note by @brainwane 2018-03-22: see [update](https://github.com/pypa/warehouse/issues/98#issuecomment-375509536)

### PR DESCRIPTION
**Pull request by jonparrott**
Closes #98

### CODE REVIEW

File: warehouse/templates/pages/help.html [hunk 1]
```diff
@@ -149,6 +151,12 @@ <h3 id="APIs">{{ APIs() }}</h3>
         <h3 id="mirroring">{{ mirroring() }}</h3>
         <p>
           If you need to run your own mirror of PyPI, the <a href="https://pypi.org/project/bandersnatch/">bandersnatch project</a> is the recommended solution. Note that the storage requirements for a PyPI mirror would exceed 1 terabyte—and growing!
+        </p>
+
+        <h3 id="notifications">{{ notifications() }}</h3>
+        <p>
+          While PyPI itself does not offer a way to get notified when new releases are uploaded, there are several third-party services that offer comprehensive monitoring and notifications for package releases and vulnerabilities such as <a href="https://pyup.io" target="_blank">pyup.io</a> and <a href="https://libraries.io/" target="_blank">libraries.io</a>.
```
Comment (alex) [inline warehouse/templates/pages/help.html]: When using `target="_blank_"` always set `rel="noopener"`

Comment (jonparrott) [reply warehouse/templates/pages/help.html]: Good catch, done in 9dfff54a.

File: warehouse/templates/pages/help.html [hunk 2]
```diff
@@ -26,6 +26,7 @@
 {% macro verified_email() %}Why do I need a verified email address?{% endmacro %}
 {% macro mirroring() %}How can I run a mirror of PyPI?{% endmacro %}
 {% macro APIs() %}Does PyPI have APIs I can use?{% endmacro %}
+{% macro notifications() %}How do I get notified when a new version of a package is released?{% endmacro %}
```
Comment (brainwane) [inline warehouse/templates/pages/help.html]: We try to use the word "project" where you say "package" here.

Comment (brainwane) [reply warehouse/templates/pages/help.html]: Could you use a more specific function name and anchor tag label? I want to futureproof here a bit; probably we will have a future FAQ item about project owners'/maintainers' email notifications.

File: warehouse/templates/pages/help.html [hunk 3]
```diff
@@ -149,6 +151,12 @@ <h3 id="APIs">{{ APIs() }}</h3>
         <h3 id="mirroring">{{ mirroring() }}</h3>
         <p>
           If you need to run your own mirror of PyPI, the <a href="https://pypi.org/project/bandersnatch/">bandersnatch project</a> is the recommended solution. Note that the storage requirements for a PyPI mirror would exceed 1 terabyte—and growing!
+        </p>
+
+        <h3 id="notifications">{{ notifications() }}</h3>
+        <p>
+          While PyPI itself does not offer a way to get notified when new releases are uploaded, there are several third-party services that offer comprehensive monitoring and notifications for package releases and vulnerabilities such as <a href="https://pyup.io" target="_blank" rel="noopener">pyup.io</a> and <a href="https://libraries.io/" target="_blank" rel="noopener">libraries.io</a>.
```
Comment (brainwane) [inline warehouse/templates/pages/help.html]: For readability I think you should split this into 2 or more sentences.

File: warehouse/templates/pages/help.html [hunk 4]
```diff
@@ -153,9 +153,9 @@ <h3 id="mirroring">{{ mirroring() }}</h3>
...
         </p>
 
-        <h3 id="notifications">{{ notifications() }}</h3>
+        <h3 id="project-release-notifications">{{ project_release_notifications() }}</h3>
         <p>
-          While PyPI itself does not offer a way to get notified when new releases are uploaded, there are several third-party services that offer comprehensive monitoring and notifications for package releases and vulnerabilities such as <a href="https://pyup.io" target="_blank" rel="noopener">pyup.io</a> and <a href="https://libraries.io/" target="_blank" rel="noopener">libraries.io</a>.
+          PyPI itself does not offer a way to get notified when a project uploads new releases. However, here are several third-party services that offer comprehensive monitoring and notifications for project releases and vulnerabilities such as <a href="https://pyup.io" target="_blank" rel="noopener">pyup.io</a> and <a href="https://libraries.io/" target="_blank" rel="noopener">libraries.io</a>.
```
Comment (brainwane) [inline warehouse/templates/pages/help.html]: Nice rewording! But "here" should be "there".

File: warehouse/templates/pages/help.html [hunk 5]
```diff
@@ -149,6 +151,12 @@ <h3 id="APIs">{{ APIs() }}</h3>
         <h3 id="mirroring">{{ mirroring() }}</h3>
         <p>
           If you need to run your own mirror of PyPI, the <a href="https://pypi.org/project/bandersnatch/">bandersnatch project</a> is the recommended solution. Note that the storage requirements for a PyPI mirror would exceed 1 terabyte—and growing!
+        </p>
+
+        <h3 id="project-release-notifications">{{ project_release_notifications() }}</h3>
+        <p>
+          PyPI itself does not offer a way to get notified when a project uploads new releases. However, here are several third-party services that offer comprehensive monitoring and notifications for project releases and vulnerabilities such as <a href="https://pyup.io" target="_blank" rel="noopener">pyup.io</a> and <a href="https://libraries.io/" target="_blank" rel="noopener">libraries.io</a>.
```
Comment (brainwane) [inline warehouse/templates/pages/help.html]: Please add comma befofe "such as".

File: warehouse/templates/pages/help.html (hunk 4 above)
Comment (jonparrott) [reply warehouse/templates/pages/help.html]: Done.

File: warehouse/templates/pages/help.html (hunk 5 above)
Comment (jonparrott) [reply warehouse/templates/pages/help.html]: It's an essential clause so I don't want to set it apart with a comma.

**Comment by di** [comment]
You're the best @jonparrott!

**Comment by jonparrott** [comment]
\o/

**Pull request merged**